public/wp-content/themes/newsup-child/home.php

```diff
@@ -60,11 +60,10 @@
                                           <?php
                                         endforeach;
                                       elseif ($post_type == 'anime_review'):
-                                        $meta_value = get_field('review_anime');
-                                        echo print_r($meta_value);
+                                        $meta_value = get_field('review_related_anime');
                                         ?>
-                                        <a class="newsup-categories category-color-1" href="<?php get_permalink( $meta_value ) ?>" alt="">
-                                          <?php esc_html( $meta_value ); ?>
+                                        <a class="newsup-categories category-color-1" href="<?php get_permalink( $meta_value->guid ) ?>" alt="">
+                                          <?php esc_html( $meta_value->post_title ); ?>
```

Comment:
For context, `line 63 $meta_value` is of type `Anime` custom post.
`$meta_value->guid` and `$meta_value->post_title` do not return data. I may be doing this wrong.
`esc_html` is just a function that escapes any potentially harmful characters when injected into the html dom.

Reply:
This code exists within a `Post Preview` content block on the Blog Post Index Page, controlled by `home.php` if the post is an `Anime` it will show `Genre(s)` in the pills, if its a `Review` it will show the single `Anime` in the pills.

Reply:
I think if you're using `get_permalink` you have to `echo` it?
From what i understand anything with `get_...` returns a string (or something similar), and anything with `the_...` prints to output, with no return.